sympy/functions/combinatorial/factorials.py

```diff
@@ -1057,3 +1058,37 @@ def _eval_is_nonnegative(self):
                 return True
             elif k.is_even is False:
                 return  False
+
+class multinomial(CombinatorialFunction):
+    r"""Implementation of the multinomial coefficient.
+    """
+    @classmethod
+    def eval(cls, *k):
+        k = [sympify(x) for x in k]
```

Comment:
```suggestion
        k = [_sympify(x) for x in k]
```

Reply:
NameError: name '_sympify' is not defined
it's throwing this ... so ...